addon/components/new-osf-navbar/component.js

```diff
@@ -34,6 +34,9 @@ export default Ember.Component.extend(hostAppName, AnalyticsMixin, {
         if (appName === 'Dummy App') {
             appName = 'Home';
         }
+        if (appName === 'Ember OSF Web') {
+            appName = window.location.pathname.indexOf('institutions') !== -1 ? 'Institutions' : 'Home';
```

Comment:
Since we'll want this to go everywhere at once, we should probably make a waffle flag for the institutional dashboard. It doesn't have to necessarily keep the page from loading, but it should keep the navbar from having the navigational option.